src/components/Form.js

```diff
@@ -135,8 +135,38 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+
+                const workspaceRegex = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                const matches = inputValue.match(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+
+                // add any html tags that are considered valid workspace names to ignoreRegxList
+                const ignoreRegxList = [/<>/, /< >/, /<->/, /<-->/, /<br>/, /<br\/>/];
+
+                let isMatch = ignoreRegxList.some(r => r.test(inputValue));
```

Comment:
My bad, I didn't clarify what I wanted here. Lets keep the ignore tags array inside the `CONST.js` and use it like:
```js
let isMatch = CONST.WHITELISTED_TAGS..some(r => r.test(inputValue));
```

Reply:
@robertKozik  no problem, PR updated